### PR TITLE
1 Synth slot every 40 pop

### PR DESCRIPTION
**Pull request by SpaceLoveSs13**
## About The Pull Request

Adds Synthetic job slot every 40 pop.

## Why It's Good For The Game

As we are constantly moving towards high pop, I believe synthetics getting more slot every 40 pop makes sense as generally there is a lack of command. Synthetics can do most shipside job, and as the pop increases, it generally gets difficult to manage shipside. Addition of a support role of Synthetic will ensure this.

I may change the pop number upon testing and feedback. This needs testing.

## Changelog
:cl:SpaceLove

balance: Gives 1 Synth job slot every 40 pop.

/:cl:

### CODE REVIEW

**Comment by lbnesquik** [comment]
God yes

File: code/datums/gamemodes/infestation.dm [hunk 1]
```diff
@@ -1 +1,9 @@
 /datum/game_mode/infestation
+
+/datum/game_mode/infestation/scale_roles()
+	
+	. = ..()
+	if(!.)
+		return
+	var/datum/job/scaled_job = SSjob.GetJobType(/datum/job/terragov/silicon/synthetic)
+	scaled_job.job_points_needed  = 40 //For every 40 marines there will be 1 extra Synth
```
Comment (BraveMole) [inline code/datums/gamemodes/infestation.dm:9]: No. This code doesn't do anything. 

You're setting that synth requires 40 job points, but you are never adding any job point to that job. 

I already told you how to do it, you either use job worth or the proc i gave you.

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/datums/gamemodes/infestation.dm (hunk 1 above)
Comment (SpaceLoveSs13) [reply code/datums/gamemodes/infestation.dm:9]: Well tivi suggested this code, and I am still unsure about your jobworth proc so I am kinda figuring it out. I am gonna make it a draft till then anyway.

File: code/datums/jobs/job/shipside.dm [hunk 2]
```diff
@@ -743,7 +755,7 @@ Use your office fax machine to communicate with corporate headquarters or to acq
 		/datum/job/terragov/squad/specialist = SPEC_POINTS_REGULAR,
 		/datum/job/terragov/squad/smartgunner = SMARTIE_POINTS_REGULAR,
 	)
-
+    job_points_needed = 40
```
Comment (BraveMole) [inline code/datums/jobs/job/shipside.dm:758]: there's an issue with identation here (too much space).

Please don't webeddit, use DM or VSC

File: code/datums/jobs/job/marines.dm [hunk 3]
```diff
@@ -451,7 +453,7 @@ You are also in charge of communicating with command and letting them know about
 	outfit = /datum/outfit/job/marine/vatgrown
 	total_positions = 0
 	job_flags = JOB_FLAG_ADDTOMANIFEST|JOB_FLAG_PROVIDES_SQUAD_HUD
-	jobworth = list(/datum/job/xenomorph = LARVA_POINTS_REGULAR)
+	jobworth = list(/datum/job/xenomorph = LARVA_POINTS_REGULAR,/datum/job/terragov/silicon/synthetic = SYNTH_POINTS_REGULAR)
```
Comment (BraveMole) [inline code/datums/jobs/job/marines.dm:456]:
```suggestion
	jobworth = list(/datum/job/xenomorph = LARVA_POINTS_REGULAR, /datum/job/terragov/silicon/synthetic = SYNTH_POINTS_REGULAR)
```

File: code/datums/jobs/job/shipside.dm [hunk 4]
```diff
@@ -51,7 +52,7 @@ Godspeed, captain! And remember, you are not above the law."})
 	belt = /obj/item/storage/belt/gun/mateba/captain/full
 	ears = /obj/item/radio/headset/mainship/mcom
 	w_uniform = /obj/item/clothing/under/marine/officer/command
-	shoes = /obj/item/clothing/shoes/marinechief/captain
+	shoes = /obj/item/clothing/shoes/marine/captain
```
Comment (BraveMole) [inline code/datums/jobs/job/shipside.dm:55]: No reason to have this

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/datums/jobs/job/shipside.dm (hunk 2 above)
Comment (SpaceLoveSs13) [reply code/datums/jobs/job/shipside.dm:758]: alright, I will do it later, thanks man.

**Pull request closed without merging**